packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -1197,7 +1197,6 @@ class _RenderTheater extends RenderBox with ContainerRenderObjectMixin<RenderBox
     while (child != null) {
       visitor(child);
       final _TheaterParentData childParentData = child.parentData! as _TheaterParentData;
-      childParentData.visitChildrenOfOverlayEntry(visitor);
```

Comment:
It took me awhile to understand this only traverse child added from overlayportal. Can you add some doc to `visitChildrenOfOverlayEntry` or maybe rename it to something like `visitOverlayPortalChildren`?